### PR TITLE
[Fix-7392][dolphinscheduler-datasource] Add hive datasource failed

### LINKED ISSUE #7392: [Bug] [DataSource] Add hive datasource failed
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

Update Kerberos environment failed when add new hive datasource.

### What you expected to happen

add new hive datasource success.

### How to reproduce

add new hive datasource.

### Anything else

It's a bug in 2.0.1-release.

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by SbloodyS**
## Purpose of the pull request

fix #7392 

## Brief change log

modify checkKerberosEnv judgment logic.

### CODE REVIEW

**Review by caishunfeng — COMMENTED** [review]
LGTM

File: dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-hive/src/main/java/org/apache/dolphinscheduler/plugin/datasource/hive/HiveDataSourceClient.java [hunk 1]
```diff
@@ -90,7 +91,8 @@ protected void checkEnv(BaseConnectionParam baseConnectionParam) {
 
     private void checkKerberosEnv() {
         String krb5File = PropertyUtils.getString(JAVA_SECURITY_KRB5_CONF_PATH);
-        if (StringUtils.isNotBlank(krb5File)) {
```
Comment (Narcasserun) [inline dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-hive/src/main/java/org/apache/dolphinscheduler/plugin/datasource/hive/HiveDataSourceClient.java:93]: Whether krb5file is empty or not may not be removed

Comment (SbloodyS) [reply dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-hive/src/main/java/org/apache/dolphinscheduler/plugin/datasource/hive/HiveDataSourceClient.java:93]: Did you mean kerberosStartupState and krb5file double check?

Comment (Narcasserun) [reply dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-hive/src/main/java/org/apache/dolphinscheduler/plugin/datasource/hive/HiveDataSourceClient.java:93]: This is the best. Krb5file does not exist, and problems are likely to occur

Comment (SbloodyS) [reply dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-hive/src/main/java/org/apache/dolphinscheduler/plugin/datasource/hive/HiveDataSourceClient.java:93]: I'll fix it.thanks.

**Comment by SbloodyS** [comment]
@caishunfeng @Narcasserun please review again. thx.

**Review by Narcasserun — APPROVED** [review]
+1

**Review by caishunfeng — APPROVED** [review]
+1

**Review by CalvinKirs — APPROVED** [review]

**Review by davidzollo — APPROVED** [review]
LGTM

**Pull request merged**